src/fable/Fable.Core/npm/fable-core.ts

```diff
@@ -1572,6 +1572,13 @@ export class Seq {
     return fst.value;
   }
 
+  static except<T>(itemsToExclude: Iterable<T>, source: Iterable<T>) {
+    const exclusionItems = Array.from(itemsToExclude);
+    const testIsNotInExclusionItems = (element: T) => !exclusionItems.some(item => item === element);
```

Comment:
Sorry, I should have told you about this. To get structural equality for unions and records, `Util.equals(item, element)` must be used instead of the `===` operator, which only compares by reference.